tests/suites/0_stateless/19_fuzz/19_0004_fuzz_filter.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+CURDIR = os.path.dirname(os.path.realpath(__file__))
+sys.path.insert(0, os.path.join(CURDIR, "../../../helpers"))
+
+from client import client
+
+log = None
+# uncomment the line below for debugging
+log = sys.stdout
+
+client1 = client(name="client1>", log=log)
+
+size = 750
+num_predicates = 500
```

Comment:
maybe shorter? The result file is too large.

Reply:
Done.